Cargo.toml

```
@@ -27,8 +27,8 @@ pin-project = "1"
 relative-path = { version = "1.3", features = ["serde"] }
 route-recognizer = "0.3"
 rustsec = "0.23"
-crates-index = "0.15" # held back by semver 0.11
-semver = { version = "0.10", features = ["serde"] } # held at 0.10 due to #63
+crates-index = "0.16"
+semver = { version = "1.0.0-rc.1", features = ["serde"] }
```

rc.2 is already in, so we might as well bump it here.

```suggestion
semver = { version = "1.0.0-rc.2", features = ["serde"] }
```

---

1.0.0 is out